### ISSUE TITLE
Recordings on by default

### ISSUE THREAD

**Issue by alexkim205** [PostHog/posthog#13948]
## Context

We'd like to turn recordings on (and console log and network request capturing) by default. This issue keeps track of all the items that must be completed before we can do this.

## Blockers

- [x] Turn on TTL for performance events table - https://github.com/PostHog/posthog/pull/13921
- [x] Add step to onboarding flow notifying user that session recordings, console logs, and network requests are turned off. @kappa90 are we on ingestion v2 right now? - https://github.com/PostHog/posthog/pull/14002

## Nice to haves

- [ ] Aggressively block out anything that looks like a token/other sensitive information in a captured network request
- [ ] Smart network request capture throttling (i.e., capturing at most X requests within a span of Y seconds)
- [ ] Granular blocklist settings for sensitive information that's captured for console logs and network requests.

## Rollout strategy

Current users with session recordings turned on
- When we release network requests, we'll eventually turn this on by default for everyone.
- Rollout network requests to top 25% of recordings users first, and followup with general release if traffic is sustainable.

Current users with session recordings turned off
- We don't want to auto turn on recordings for these users because they may have very good reason to not have turned it on. 
- However we can still keep the quick start setup option for people who haven't set it up yet.

New users
- Turn on everything by default in new onboarding flow

Tagging @benjackwhite @annikaschmid @pjhul @kappa90 for discussion and thoughts in general

**Comment by kappa90** [comment]
> Add step to onboarding flow notifying user that session recordings, console logs, and network requests are turned off. @kappa90 are we on ingestion v2 right now?

It's still behind a feature flag but it's there. @raquelmsmith was planning on adding something wrt recordings and autocapture in the onboarding flow btw

Wouldn't turning recordings on by default be a potential problem with sensitive information leakage?

**Comment by raquelmsmith** [comment]
Yeah we discussed this with Tim and James in a growth review recently. The conclusion (feel free to disagree) was to _not_ turn them on by default because some people can be very anti due to privacy concerns. But, as @kappa90 mentioned we would add a step in onboarding to explicitly ask them if they want to turn it on.

**Comment by benjackwhite** [comment]
I think the idea was to have a very blocking step in the onboarding for new projects like my markdown mockup below (imagine it being a pretty step in the process)

That way we only turn it on _after_ that step where they have seen the modal and they press "Done". So it is still opt-in but we're helping them there. We have plenty of customers who are like "why don't I have recordings?" And then are frustrated to find out they had to go to a different place to turn them on.

---
## Session Recordings are Enabled!

> Blurb about why recordings are great

- [x] Recordings Enabled
- [x] Log recording is Enabled
- [x] Network recording is Enabled

`Disable Session Recording`  -  `Done`

---

**Comment by raquelmsmith** [comment]
Ok I'm fine with the checkbox being auto-selected as long as they have to explicitly approve it!
